packages/graphiql/src/components/GraphiQL.tsx

```diff
@@ -714,197 +721,205 @@ export class GraphiQL extends React.Component<GraphiQLProps, GraphiQLState> {
     };
 
     return (
-      <div
-        ref={n => {
-          this.graphiqlContainer = n;
-        }}
-        className="graphiql-container">
-        {this.state.historyPaneOpen && (
-          <div className="historyPaneWrap" style={historyPaneStyle}>
-            <QueryHistory
-              ref={node => {
-                this._queryHistory = node;
-              }}
-              operationName={this.state.operationName}
-              query={this.state.query}
-              variables={this.state.variables}
-              onSelectQuery={this.handleSelectHistoryQuery}
-              storage={this._storage}
-              maxHistoryLength={this.state.maxHistoryLength}
-              queryID={this._editorQueryID}>
-              <button
-                className="docExplorerHide"
-                onClick={this.handleToggleHistory}
-                aria-label="Close History">
-                {'\u2715'}
-              </button>
-            </QueryHistory>
-          </div>
-        )}
-        <div className="editorWrap">
-          <div className="topBarWrap">
-            <div className="topBar">
-              {logo}
-              <ExecuteButton
-                isRunning={Boolean(this.state.subscription)}
-                onRun={this.handleRunQuery}
-                onStop={this.handleStopQuery}
-                operations={this.state.operations}
-              />
-              {toolbar}
-            </div>
-            {!this.state.docExplorerOpen && (
-              <button
-                className="docExplorerShow"
-                onClick={this.handleToggleDocs}
-                aria-label="Open Documentation Explorer">
-                {'Docs'}
-              </button>
-            )}
-          </div>
-          <div
-            ref={n => {
-              this.editorBarComponent = n;
-            }}
-            className="editorBar"
-            onDoubleClick={this.handleResetResize}
-            onMouseDown={this.handleResizeStart}>
-            <div className="queryWrap" style={queryWrapStyle}>
-              <QueryEditor
-                ref={n => {
-                  this.queryEditorComponent = n;
+      <>
```

Comment:
I don't think we need to wrap the entire output in a fragment. I suspect this is why there is such a huge diff in this render!? would like to simplify this diff so I can understand these changes better